src/pattern-library/components/LibraryNext.js

```diff
@@ -57,6 +57,35 @@ function ChangelogItem({ status, children }) {
   );
 }
 
+/**
+ *
+ * @param {object} props
+ *   @param {string} props.content
+ *   @param {'sm'|'md'|'lg'} [props.size]
+ *   @param {string} [props.title]
+ */
+function Code({ content, size, title }) {
+  return (
+    <div className="space-y-2">
+      <div
+        className={classnames(
+          'unstyled-text bg-slate-7 text-color-text-inverted p-4 rounded-md',
+          { 'text-sm': size === 'sm' }
+        )}
+      >
```

Comment:
What is the reason for the wrapper `<div>` as opposed to styling the `<code>` itself?

Reply:
`code` is an inline element, so it felt more appropriate to have a block container around it here.